doc/source/serve/doc_code/create_deployment.py

```diff
@@ -43,81 +42,59 @@ def __init__(self, path: str) -> None:
-# prediction  : (pid: 62637); path: /model/rep-2.pkl; data: 0.751; prediction: 1.444
-# --
-# HTTP responses: ----------
-# handle name : rep-1
-# prediction  : (pid: 62636); path: /model/rep-1.pkl; data: 0.582; prediction: 1.481
-# handle name : rep-2
-# prediction  : (pid: 62637); path: /model/rep-2.pkl; data: 0.778; prediction: 1.678
-# handle name : rep-1
-# prediction  : (pid: 62634); path: /model/rep-1.pkl; data: 0.139; prediction: 0.139
-# handle name : rep-2
-# prediction  : (pid: 62635); path: /model/rep-2.pkl; data: 0.569; prediction: 1.262
+    # our request.
+    async def predict(self, data: float) -> str:
+        pred = self.model.predict(data)
+        return (f"(pid: {self.pid}); path: {self.path}; "
+                f"data: {float(data):.3f}; prediction: {pred:.3f}")
+
+    async def __call__(self, http_request: starlette.requests.Request) -> str:
+        data = float(await http_request.query_params['data'])
+        return await self.predict(data)
+
+
+@serve.deployment
+class ServeHandleDemo:
+    def __init__(self, predictor_1: Predictor, predictor_2: Predictor):
+        self.predictor_1 = predictor_1
+        self.predictor_2 = predictor_2
+
+    async def run(self):
+        for i in range(2):
+            for predictor in [self.predictor_1, self.predictor_2]:
+                # Call our deployments from Python using the ServeHandle API.
+                random_prediction = await predictor.predict.remote(random())
+                print(f"prediction: {random_prediction}")
+
+    async def __call__(self, http_request: starlette.requests.Request) -> str:
+        return await self.run()
+
+
+# Connect to an existing Ray cluster.  Make sure to start a Ray cluster before
+# running this script to avoid `ConnectionError: Could not find any running Ray instance.`
+ray.init(address="auto")
+
+rep_1_predictor = Predictor.options(
+    name="rep_1", num_replicas=2).bind("/model/rep-1.pkl")
+rep_2_predictor = Predictor.options(
+    name="rep_2", num_replicas=2).bind("/model/rep-2.pkl")
+
+# Pass in our deployments as arguments.  At runtime, these are resolved to ServeHandles.
+serve_handle_demo = ServeHandleDemo.bind(rep_1_predictor, rep_2_predictor)
+
+serve.run(serve_handle_demo)
+
+print("ServeHandle API responses: " + "--" * 5)
+
+url = f"http://127.0.0.1:8000/"
+response = requests.get("http://127.0.0.1:8000/")
+prediction = response.text
+print(f"prediction : {prediction}")
+
+# Output (info logs omitted for brevity):
+
+# (ServeReplica:ServeHandleDemo pid=92100) prediction: (pid: 92096); path: /model/rep-1.pkl; data: 0.721; prediction: 1.442
+# (ServeReplica:ServeHandleDemo pid=92100) prediction: (pid: 92099); path: /model/rep-2.pkl; data: 0.204; prediction: 0.204
+# (ServeReplica:ServeHandleDemo pid=92100) prediction: (pid: 92097); path: /model/rep-1.pkl; data: 0.669; prediction: 1.390
+# (ServeReplica:ServeHandleDemo pid=92100) prediction: (pid: 92098); path: /model/rep-2.pkl; data: 0.791; prediction: 1.511
```

Comment:
I think I'm using the API correctly, but I still get these warnings in the output:

```
(ServeReplica:ServeHandleDemo pid=93034) /Users/archit/ray/python/ray/serve/handle.py:310: UserWarning: From /Users/archit/ray/python/ray/serve/handle.py:310: get_deployment (from ray.serve.api) is deprecated and will be removed in a future version Please see https://docs.ray.io/en/latest/serve/index.html

(ServeReplica:ServeHandleDemo pid=93034) You are retrieving a sync handle inside an asyncio loop. Try getting Deployment.get_handle(.., sync=False) to get better performance. Learn more at https://docs.ray.io/en/latest/serve/handle-guide.html#sync-and-async-handles
```

Is this a known issue?  If not I can create an issue for it

Reply:
please create an issue with simple repro. this should be release blocking for Serve

Reply:
`UserWarning` no longer appears on master.

Created an issue for the sync handle message here: https://github.com/ray-project/ray/issues/27266